exercises/series/canonical-data.json

```diff
@@ -0,0 +1,134 @@
+{
+  "exercise": "series",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "slices of one from one",
+      "property": "slices",
+      "input": {
+        "series": "1",
+        "sliceLength": 1
+      },
+      "expected": ["1"]
+    },
+    {
+      "description": "slices of one from two",
+      "property": "slices",
+      "input": {
+        "series": "12",
+        "sliceLength": 1
+      },
+      "expected": ["1", "2"]
+    },
+    {
+      "description": "slices of two",
+      "property": "slices",
+      "input": {
+        "series": "12",
+        "sliceLength": 2
+      },
+      "expected": ["12"]
+    },
+    {
+      "description": "slices of two with nonsequential numbers",
+      "property": "slices",
+      "input": {
+        "series": "35",
+        "sliceLength": 2
+      },
+      "expected": ["35"]
+    },
```

Comment:
Since we are dealing with characters and not numbers / digits how is "slices of two" any different than "slices of two with nonsequential numbers"?

Reply:
I don't think this test case adds anything over the previous one.